panther_hardware_interfaces/include/panther_hardware_interfaces/roboteq_driver.hpp

```diff
@@ -0,0 +1,198 @@
+// Copyright 2023 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef PANTHER_HARDWARE_INTERFACES_ROBOTEQ_DRIVER_HPP_
+#define PANTHER_HARDWARE_INTERFACES_ROBOTEQ_DRIVER_HPP_
+
+#include <atomic>
+#include <condition_variable>
+#include <cstdint>
+#include <vector>
+
+#include <lely/coapp/fiber_driver.hpp>
+
+#include <panther_msgs/msg/fault_flag.hpp>
+#include <panther_msgs/msg/runtime_error.hpp>
+#include <panther_msgs/msg/script_flag.hpp>
```

Comment:
```suggestion
#include <atomic>
#include <chrono>
#include <condition_variable>
#include <cstdint>
#include <memory>
#include <mutex>
#include <string>

#include <lely/coapp/fiber_driver.hpp>
```
Some are not used, some are missing